src/backend/distributed/shared_library_init.c

```diff
@@ -483,14 +482,12 @@ RegisterCitusConfigVariables(void)
 	DefineCustomEnumVariable(
 		"citus.task_executor_type",
 		gettext_noop("Sets the executor type to be used for distributed queries."),
-		gettext_noop("The master node chooses between three different executor types "
-					 "when executing a distributed query. The router executor is "
-					 "optimal for simple key-value lookups on a single shard. The "
-					 "real-time executor is optimal for queries that involve "
-					 "aggregations and/or co-located joins on multiple shards. The "
-					 "task-tracker executor is optimal for long-running, complex "
-					 "queries that touch thousands of shards and/or that involve "
-					 "table repartitioning."),
+		gettext_noop("The master node chooses between two different executor types "
```

Comment:
Why we removed router executor at all? Shouldn't we also mention that real-time executor is optimal for key-value lookups on a single shard?